src/js/components/Form/stories/ArrayOfFormFields.js

```diff
@@ -55,7 +55,9 @@ export const ArrayOfFormFields = () => {
               return undefined;
             },
           ]}
-        />
+        >
+          <TextInput aria-label="phone number" name="phone number" type="tel" />
```

Comment:
Instead of aria-label should we add `htmlFor` to the FormField and an `id` to the TextInput so that the TextInput is linked to the FormField label?